### PR TITLE
Feature/investify bootstrapping

### PR DESCRIPTION
**Pull request by gbonumore**
After https://github.com/pie-dao/monorepo/pull/44 goes through this can be revised too

### CODE REVIEW

File: .lintstagedrc [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
-  "*.{none}": ["eslint --fix", "prettier --write"]
```
Comment (adam-arold) [inline .lintstagedrc:2]: we should at some point change `none` to `ts,js`, etc and fix the errors 😅  Let's decide when can we do this!

**Review by adam-arold — APPROVED** [review]
Looks solid. I have a small comment, but that's all. 🚀

File: .lintstagedrc (hunk 1 above)
Comment (gbonumore) [reply .lintstagedrc:2]: let's do it, the sooner the better

Comment (adam-arold) [reply .lintstagedrc:2]: let's get the current PRs merged then I'll open a new one with the fixes 👀

**Review by jordaniza — APPROVED** [review]
Assuming the storybook changes drop out, the NextJS implementation looks perfectly sensible to me.

**Pull request merged**